src/http/headers.cr

```diff
@@ -222,4 +230,16 @@ struct HTTP::Headers
       values.join ","
     end
   end
+
+  private def check_invalid_header_content(value)
+    # According to RFC 7230, characters accepted as HTTP header
+    # are '\t', ' ', all US-ASCII printable characters and
+    # range from '\x80' to '\xff' (but the last is obsoleted.)
+    value.each_char do |char|
```

Comment:
It is not easy because multi-byte character shouldn't accept. However, if given string is acceptable, this string is `single_byte_optimized`, so performance issue is nothing.